mocks/mutators.json

```diff
@@ -0,0 +1,5 @@
+{
+    " break; ": [" continue; "],
+    " return std::all_of(m_addr.begin(), m_addr.end(), [](uint8_t b) { return b == 0; }); ": [" return std::any_of(m_addr.begin(), m_addr.end(), [](uint8_t b) { return b == 0; }); "],
```

Comment:
Shouldn't it also create the following mutation?

` return std::all_of(m_addr.begin(), m_addr.end(), [](uint8_t b) { return b != 0; }); ` (the `!=`)

Reply:
Yes, you're right! Nice catch!